### PR TITLE
[CORE-213] - added language picker directive to main templates and ad…

### PR DESCRIPTION
**Pull request by TolgaCura**
…ded simple styling to picker

### CODE REVIEW

**Comment by TolgaCura** [comment]
I have created a test branch for this in the demo-widget project, please see branch CORE-213 on gitlab

**Comment by TolgaCura** [comment]
screen shot 2016-12-15 at 11 20 01
screen shot 2016-12-15 at 11 20 33
screen shot 2016-12-15 at 11 21 19
screen shot 2016-12-15 at 11 21 38
screen shot 2016-12-15 at 11 23 10
screen shot 2016-12-15 at 11 23 30
screen shot 2016-12-15 at 11 23 53

File: src/admin-booking/templates/main.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
   <div bb-form-data-store></div>
 
+  <div bb-language-picker></div>
```
Comment (lukeferguson) [inline src/admin-booking/templates/main.html]: is the language picker hidden when there's only one available language?

File: src/core/templates/i18n/language_picker.html [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
-           ng-change="vm.pickLanguage(vm.language.selected.identifier)" enable-search="false">
-  <ui-select-match placeholder="Select..."><span translate="{{$select.selected.label}}"></span></ui-select-match>
-  <ui-select-choices repeat="lang in vm.availableLanguages | props: { label: $select.search}">
-    <div ng-bind-html="lang.label | translate | highlight: $select.search"></div>
-  </ui-select-choices>
-</ui-select>
\ No newline at end of file
+<div id="language-picker-container" class="container col-xs-12">
+  <ui-select id="language-picker" ng-model="vm.language.selected" theme="bootstrap"
+             ng-change="vm.pickLanguage(vm.language.selected.identifier)" enable-search="false"
```
Comment (lukeferguson) [inline src/core/templates/i18n/language_picker.html]: Search appears to be enabled in the screenshots? Maybe we can clever about this and only enable it when there's more than something like 10 options?

File: src/core/templates/i18n/language_picker.html [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
+             class="pull-right col-xs-12 col-sm-3 col-md-2 col-lg-2">
```
Comment (lukeferguson) [inline src/core/templates/i18n/language_picker.html]: Enforcing styling is potentially a bit too prescriptive - does this work for studio too? might be better to wrap the component with this instead

File: src/core/templates/i18n/language_picker.html [hunk 4]
```diff
@@ -1,7 +1,10 @@
-<ui-select ng-model="vm.language.selected" theme="bootstrap"
-           ng-change="vm.pickLanguage(vm.language.selected.identifier)" enable-search="false">
-  <ui-select-match placeholder="Select..."><span translate="{{$select.selected.label}}"></span></ui-select-match>
-  <ui-select-choices repeat="lang in vm.availableLanguages | props: { label: $select.search}">
-    <div ng-bind-html="lang.label | translate | highlight: $select.search"></div>
-  </ui-select-choices>
-</ui-select>
\ No newline at end of file
+<div id="language-picker-container" class="container col-xs-12">
```
Comment (lukeferguson) [inline src/core/templates/i18n/language_picker.html]: container classes should only ever appear once in the document

File: src/admin-booking/templates/main.html (hunk 1 above)
Comment (TolgaCura) [inline src/admin-booking/templates/main.html]: the directive for the language picker handles showing/hiding the component
should this logic be implemented on the view?

see line 17: https://github.com/BookingBug/bookingbug-angular/blame/master/src/core/javascripts/i18n/directives/language_picker.directive.js.coffee

**Comment by TolgaCura** [comment]
Updated PR
- removed all styling
- Search enabled when 10+ languages
- enabled placeholder translation (displays 'Select...' translation key if no translation present)

**Comment by TolgaCura** [comment]
screen shot 2016-12-22 at 11 35 57
screen shot 2016-12-22 at 11 36 30
screen shot 2016-12-22 at 11 37 02
screen shot 2016-12-22 at 11 37 16
screen shot 2016-12-22 at 11 38 08

**Pull request merged**